src/components/Books.js

```diff
@@ -0,0 +1,8 @@
+const Books = () => (
+  <div>
```

Comment:
- [ ] Good job adding this default component, from the instructions, but this component should have a button for removing a book, though not yet functional. You should also have placeholder data for books which should be made available to a child component through props.
- [ ] kindly create another component called book that will render books which it should receive through props, this is part of the project instructions.

Reply:
Done, thanks for your feadback